src/app/backend/resource/generic/controller.go

```diff
@@ -214,3 +256,26 @@ func (self StatefulSetController) Get(allPods []v1.Pod, allEvents []v1.Event) Re
 func (self StatefulSetController) UID() types.UID {
 	return apps.StatefulSet(self).UID
 }
+
+// GetLogSources is an implementation of the GetLogSources method from ResourceController interface.
+func (self StatefulSetController) GetLogSources(k8sClient *client.Clientset) LogSources {
+	return LogSources{
+		PodNames:       getPodNames(k8sClient, self.Namespace, self.UID()),
+		ContainerNames: common.GetContainerNames(&self.Spec.Template.Spec),
+	}
+}
+
+func getPodNames(k8sClient *client.Clientset, namespace string, uid types.UID) []string {
+	allPods, _ := k8sClient.CoreV1().Pods(namespace).List(listEverything)
+	matchingPods := common.FilterPodsByOwnerReference(namespace, uid, allPods.Items)
+	names := make([]string, 0)
+	for _, pod := range matchingPods {
+		names = append(names, pod.Name)
+	}
+	return names
+}
+
+var listEverything = meta.ListOptions{
```

Comment:
Also, I think we may use this variable somewhere else. You could make it public if it is true and simplify code.